src/sass/app_2/_common/base/constants.scss

```diff
@@ -10,24 +10,30 @@ $COLOR_PURPLE     : #4f60ae;
 $COLOR_RED        : #e31c4b;
 $COLOR_SKY_BLUE   : #2196f3;
 $COLOR_WHITE      : #ffffff;
+$COLOR_BLACK      : #000000;
 // Light theme
-$COLOR_LIGHT_BLACK_1      : rgba(0, 0, 0, 0.8);
-$COLOR_LIGHT_BLACK_2      : rgba(0, 0, 0, 0.4);
-$COLOR_LIGHT_BLACK_3      : rgba(0, 0, 0, 0.16);
-$COLOR_LIGHT_BLACK_4      : rgba(0, 0, 0, 0.04);
-$COLOR_LIGHT_BLACK_4_SOLID: #f4f4f6;
-$COLOR_LIGHT_GRAY_1       : #999cac;
-$COLOR_LIGHT_GRAY_2       : rgba(153, 156, 172, 0.32);
+$COLOR_LIGHT_BLACK_1       : rgba(0, 0, 0, 0.8);
+$COLOR_LIGHT_BLACK_2       : rgba(0, 0, 0, 0.4);
+$COLOR_LIGHT_BLACK_3       : rgba(0, 0, 0, 0.16);
+$COLOR_LIGHT_BLACK_4       : rgba(0, 0, 0, 0.04);
+$COLOR_LIGHT_BLACK_4_SOLID : #f4f4f6;
+$COLOR_LIGHT_GRAY_1        : #999cac;
+$COLOR_LIGHT_GRAY_2        : rgba(153, 156, 172, 0.32);
+$COLOR_LIGHT_GREEN_GRADIENT: linear-gradient(to top, rgba(255, 255, 255, 0), rgba(0, 148, 117, 0.16));
+$COLOR_LIGHT_RED_GRADIENT  : linear-gradient(to top, rgba(255, 255, 255, 0), rgba(227, 28, 75, 0.16));
+$COLOR_LIGHT_WHITE_GRADIENT: linear-gradient(to right, rgba(255, 255, 255, 1), rgba(255, 255, 255, 0));
 // Dark theme
-$COLOR_DARK_BLUE_1: #0b0e18;
-$COLOR_DARK_BLUE_2: #101320;
-$COLOR_DARK_BLUE_3: #191c31;
-$COLOR_DARK_BLUE_4: #202641;
-$COLOR_DARK_BLUE_5: #2a3052;
-$COLOR_DARK_BLUE_6: #555975;
-$COLOR_DARK_BLUE_7: #7f8397;
-$COLOR_DARK_BLUE_8: rgba(127, 131, 151, 0.3);
-
+$COLOR_DARK_BLUE_1         : #0b0e18;
+$COLOR_DARK_BLUE_2         : #101320;
+$COLOR_DARK_BLUE_3         : #191c31;
+$COLOR_DARK_BLUE_4         : #202641;
+$COLOR_DARK_BLUE_5         : #2a3052;
+$COLOR_DARK_BLUE_6         : #555975;
+$COLOR_DARK_BLUE_7         : #7f8397;
+$COLOR_DARK_BLUE_8         : rgba(127, 131, 151, 0.3);
+$COLOR_DARK_GREEN_GRADIENT : linear-gradient(to top, rgba(16, 19, 32, 0), rgba(0, 148, 117, 0.16));
+$COLOR_DARK_RED_GRADIENT   : linear-gradient(to top, rgba(16, 19, 32, 0), rgba(227, 28, 75, 0.16));
+$COLOR_DARK_BLACK_GRADIENT : linear-gradient(to right, rgba(16, 19, 32, 1), rgba(16, 19, 32, 0));
```

Comment:
should be on line 26, or if you want to keep all gradients together then on line 34